games/ninswitch/game_imagerec.py

```diff
@@ -31,10 +33,28 @@
     ((201, 650), (2, 2, 4)),
 ]
 
+class capture_screen(Switch):
```

Comment:
I think this could be more elegant by having this class just flipping a new global variable `save_frame` between True/False, and sharing the same cap with `image_rec_main`. (also renaming SAVE_FRAMES --> SAVE_ALL_FRAMES).

Then in `image_rec_main` I would update:
```python
if SAVE_ALL_FRAMES or save_frame:
    cv2.imwrite(f"{SAVE_DIR_PATH}/{i}.jpg", frame)
    logging.info(f"SAVED {i}.jpg")
```
What do you think, does it make sense?

Reply:
Also PEP8 naming: capture_screen --> CaptureScreen
I don't think this is enforced by our pipelines yet, but soon it might :D

Reply:
I tried to do something like this and was unable to get it to work. You might be able to get it to work by using `global`.